llvm/utils/llvm_push_pr.py

```diff
@@ -0,0 +1,653 @@
+#!/usr/bin/env python3
+"""A script to automate the creation and landing of a stack of Pull Requests."""
+
+import argparse
+import os
+import re
+import subprocess
+import sys
+import time
+from typing import List, Optional
+
+import requests
```

Comment:
Is there a reason to prefer `requests` (third party library) over the stdlib `urllib.request`?

Reply:
Moved to use just things from the sandard lib.